torch/onnx/symbolic_opset9.py

```diff
@@ -1415,28 +1415,11 @@ def bitwise_not(g, inp):
     return g.op("Not", inp)
 
 
-def wrap_logical_op_with_cast_to(to_type):
-    def decorator(fn):
-        def wrap_with_cast(g, input, other):
-            return g.op(
-                "Cast",
-                fn(g, input, other),
-                to_i=symbolic_helper.cast_pytorch_to_onnx[to_type],
-            )
-
-        return wrap_with_cast
-
-    return decorator
-
-
 def wrap_logical_op_with_cast_to_and_from(to_type):
```

Comment:
I think this decorator and its usage can all be removed. Casting to bool are automatically handled in `scalar_type_analysis.cpp` already I believe.

Reply:
You are right, I got confused with the `onnx::Greater` that were actually handled in the mentioned pass.
Please rename the decorator, as now it is no longer casting results.